src/chericat.c

```diff
@@ -54,25 +55,30 @@
 
 static void usage()
 {
-	fprintf(stderr, "Usage: chericat [-d <debug level>] [-f <database name>] [-p <pid>] [-v]\n\t[-c <binary name>]\n"
-			"     debug level - 0 = No output; 1 = INFO; 2 = VERBOSE; 3 = TROUBLESHOOT\n"
-			"     pid - pid of the target process for a snapshot of caps info\n"
+	fprintf(stderr, "Usage: chericat [-d*] [-f <database name>] [-p <pid>] [-v]\n\t[-c <binary name>]\n"
```

Comment:
Not user what `*` is supposed to mean. You usually just specify that -d can be passed and say that additional times increases debugging.

Reply:
Fixed - removed the * (it was supposed to represent zero or more) and reduced the details on the usage message. Also squashed the previous 5 commits.

Thanks!

Reply:
Thanks for the suggestion, committed.